apps/builder/app/shared/copy-paste/plugin-instance.ts

```diff
@@ -204,32 +199,10 @@ export const onPaste = (clipboardData: string): boolean => {
     if (newRootInstanceId === undefined) {
       return;
     }
-    let dropTarget = pasteTarget;
-    dropTarget =
-      getInstanceOrCreateFragmentIfNecessary(data.instances, dropTarget) ??
-      dropTarget;
-    dropTarget =
-      wrapEditableChildrenAroundDropTargetMutable(
-        data.instances,
-        data.props,
-        metas,
-        dropTarget
-      ) ?? dropTarget;
-    const [parentId] = dropTarget.parentSelector;
-    const parentInstance = data.instances.get(parentId);
-    if (parentInstance === undefined) {
-      return;
-    }
-    const child: Instance["children"][number] = {
-      type: "id",
-      value: newRootInstanceId,
-    };
-    const { position } = dropTarget;
-    if (position === "end") {
-      parentInstance.children.push(child);
-    } else {
-      parentInstance.children.splice(position, 0, child);
-    }
+    const children: Instance["children"] = [
+      { type: "id", value: newRootInstanceId },
+    ];
+    addInstanceChildrenMutable(data, children, pasteTarget);
```

Comment:
add is usually considered "at the end" semantically, I think "insert" is more correct

Reply:
if this is also inserting at any position